### PR TITLE
Upgrade to v1.0.0

### PR DESCRIPTION
**Pull request by alfonsomunozpomer**
I think this module has come of age, it’s time to bump its major version number.

The scripts that generate the fixtures have changed significantly: rather than start by a random set of cell IDs and gene IDs and hope there would be marker genes and so on, we start now from a set of cell groups, pick some marker genes there, and continue forward with 50 random cell IDs where there are marker genes. We also ensure that an `inferred cell type` variable is chosen in each experiment.

@pmb59 Could you please have a look at the new `README.md` to ensure it’s all accurate and up-to-date? I think you can skip the fixture-generating scripts since that’s something a developer should review, but feel free to have a look, of course.

### CODE REVIEW

File: README.md
Comment (upendrakumbham) [inline README.md]: I think it is Postgres 11.

File: README.md
Comment (upendrakumbham) [inline README.md]: `ths` I think typo 'this'

File: README.md
Comment (upendrakumbham) [inline README.md]: `postgres 10 database` here refactor to Postgres 11

File: README.md
Comment (upendrakumbham) [inline README.md]: `in[the` give a space after in

File: README.md
Comment (upendrakumbham) [inline README.md]: `such as "umap" or "tsne"; the value is arbitrary and supplied by the user`, not sure we need to mention in the capitals 'UMPA' and 't-SNE' as per the database.

File: README.md
Comment (upendrakumbham) [inline README.md]: a bit confusing here. loading and reading the table..etc

File: README.md
Comment (alfonsomunozpomer) [reply README.md]: Since the value is arbitrary there’s no need to restrict it to lowercase. I’ll include your suggestion.

File: fixtures/generate-fixtures.sh
Comment (upendrakumbham) [inline fixtures/generate-fixtures.sh]: just typo `_croup` instead of` group`

File: fixtures/generate-tsv-fixture.sh
Comment (upendrakumbham) [inline fixtures/generate-tsv-fixture.sh]: typo here - `differnet` - different

File: fixtures/README.md
Comment (upendrakumbham) [inline fixtures/README.md]: remove extra` t` from here `/tes/tresources/`

File: fixtures/generate-fixtures.sh
Comment (upendrakumbham) [inline fixtures/generate-fixtures.sh:10]: Above experiment, accessions are a bit confusing, why we added -  17, 24 19..etc to the existing experiment accessions.

**Review by upendrakumbham — APPROVED** [review]
Repo README looks good. Please fix minor comments in the fixtures and its README before merging this PR.

File: README.md
Comment (pmb59) [inline README.md]: I could not find release v0.4.0 anywhere

File: README.md
Comment (pmb59) [inline README.md]: you could explicitly set the minimum version of R required here

File: README.md
Comment (pmb59) [inline README.md]: I don't understand what what `wide format` means here. Also, what is `iRAP SC`? We only use IRAP for bulk analysis in Atlas

if possible, can we give here the prefix/suffix/extension of the EXPERIMENT_CLUSTERS_FILEs?

File: README.md
Comment (pmb59) [inline README.md]: what's `ISL` here? We use `scanpy` for single-cell in Atlas

ISL/iRAP are related to bulk only.

File: README.md
Comment (pmb59) [inline README.md]: if possible, can we give here the prefix/suffix/extension of the marker gene files?

File: README.md
Comment (pmb59) [inline README.md]: Do you mean GitHub actions ? (instead of Travis)

File: README.md
Comment (pmb59) [inline README.md]: tag `0.2.1` of the container is 4 years old (same as `0.2.0`). Are you sure that the requirement `or any of the tags after 0.2.0` is correct?

File: README.md
Comment (pmb59) [inline README.md:51]: if possible, can we give here the prefix/suffix of the coordinate files?

File: README.md
Comment (pmb59) [inline README.md:118]: any reason regarding why this info has been removed?

File: fixtures/generate-fixtures.sh
Comment (pmb59) [reply fixtures/generate-fixtures.sh:10]: +1

**Review by pmb59 — CHANGES_REQUESTED** [review]
Hi @alfonsomunozpomer  I've left few minor comments about the README

File: README.md
Comment (alfonsomunozpomer) [reply README.md]: Since 0.2.3 was used for the May 2020 release, and versioning of this package had been neglected for a while, I thought that the version used in October 2022 would have been something around 0.4.0.

File: README.md
Comment (alfonsomunozpomer) [reply README.md]: I don’t know which is the minimum version, this is something set by Data Production and you. Feel free to modify it.

File: README.md
Comment (alfonsomunozpomer) [reply README.md]: My guess is that iRAP SC is a variant of [iRAP](https://www.biorxiv.org/content/10.1101/005991v1) for single cell sequencing. I left that part as-is. I just reformatted it as a table.

Comment (alfonsomunozpomer) [reply README.md]: The files are the `<EXP-ID>.clusters.tsv` files. The name is arbitrary, it could very well be `foobar.txt` as long as the format is correct.

File: README.md
Comment (alfonsomunozpomer) [reply README.md]: At some point the clusters format was `ISL` but we’ve been using `SCANPY` for a while. However, I don’t know the internals of the script and I copy-pasted this.

File: README.md
Comment (alfonsomunozpomer) [reply README.md]: This is a directory name, so there’s no need for that.

File: README.md
Comment (alfonsomunozpomer) [reply README.md]: Yes, I guess this is also legacy documentation that no longer applies.

File: README.md
Comment (alfonsomunozpomer) [reply README.md]: Since I’m not the mantainer I don’t know how up-to-date it is. Feel free to publish the container and fix this.

File: README.md
Comment (alfonsomunozpomer) [reply README.md:118]: My mistake, I’ll revert it.

File: README.md
Comment (alfonsomunozpomer) [reply README.md:51]: There’s no prefix/suffix, `DIMRED_FILE_PATH` points at a specific file. It can be any name.

File: README.md
Comment (pmb59) [reply README.md]: current version is R 3.6.3

File: README.md
Comment (pmb59) [reply README.md]: I've not been the maintainer neither, let's just recommend 'latest'

File: README.md
Comment (pmb59) [reply README.md:118]: thanks

**Review by pmb59 — APPROVED** [review]

File: fixtures/generate-fixtures.sh
Comment (upendrakumbham) [reply fixtures/generate-fixtures.sh:10]: Alfonso, could you please clarify the above changes, please?

File: fixtures/generate-fixtures.sh
Comment (upendrakumbham) [reply fixtures/generate-fixtures.sh]: this one also, please.

**Review by upendrakumbham — CHANGES_REQUESTED** [review]
Please fix those minor typos in the README.

File: fixtures/generate-fixtures.sh
Comment (alfonsomunozpomer) [reply fixtures/generate-fixtures.sh:10]: It’s now explained in the README file of the fixtures directory.

**Review by upendrakumbham — APPROVED** [review]
Thanks for the clarifications. All looks good to me.

**Pull request merged**
